scripts/hail_batch/eqtl_hail_batch/launch_generate_eqtl_spearman.py

```diff
@@ -0,0 +1,62 @@
+"""Launch analysis runner for all cell types and chromosomes"""
+
+import re
+import os
+from analysis_runner.cli_analysisrunner import run_analysis_runner
+from google.cloud import storage
+
+REFERENCE = 'GRCh38'
+BUCKET_NAME = 'cpg-tob-wgs-test'
+INPUT_PATH = f'gs://{BUCKET_NAME}/full_data/{REFERENCE.lower()}/analysis_output/kccg'
```

Comment:
I just tested this using the structure that's in the kccg bucket. I imaging it'll look similar to this when Seyhan uploads it